tests/tests/tier0/monitor-node-disconnect/python/monitor.py

```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+from dasbus.loop import EventLoop
+
+from bluechi.api import Manager, Node
+
+
+loop = EventLoop()
```

Comment:
Best practices - Python scripts should, in most cases, be protected from running code unintentionally (e.g. when the file is imported). So, it is better to move all the code into a `main` function and run it only if the script is actually executed:

```python
def main():
  loop = EventLoop()
  ...
  ...


if __name__ == "__main__":
  main()
```

Reply:
True. Changed it accordingly.